src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenderSubAccount.java

```diff
@@ -36,7 +36,6 @@ public class OffenderSubAccount {
     @Column(name = "TRUST_ACCOUNT_CODE", nullable = false, insertable = false, updatable = false)
     private Long accountCode;
 
-    @Id
```

Comment:
Not actually included in the id and causes an error in latest Hibernate verison.